### PR TITLE
bugfix/ATC-241

### LINKED ISSUE #241: Arrivals' landings cause "waiting" departures to be deleted
See zlsa#770.

### PR DESCRIPTION
**Pull request by erikquinn**
Fixes #241.

### CODE REVIEW

File: src/assets/scripts/airport/RunwayModel.js [hunk 1]
```diff
@@ -127,13 +129,9 @@ export default class RunwayModel extends BaseModel {
...
-    removeQueue(aircraft, force) {
-        if (this.queue[0] === aircraft || force) {
-            this.queue.shift(aircraft);
-
-            if (this.queue.length >= 1) {
-                this.queue[0].moveForward();
-            }
+    removeQueue(aircraft) {
+        if (_has(this.queue, aircraft)) {
+            this.queue = _without(this.queue, aircraft);
```
Comment (n8rzz) [inline src/assets/scripts/airport/RunwayModel.js:134]: I love lodash!  This is so much easier to read!! 👍

File: src/assets/scripts/airport/AirportController.js [hunk 2]
```diff
@@ -200,8 +200,8 @@ export default class AirportController {
         const runwaySecondaryEndIndex = 1;
         const runways = this.airport_get().runways;
         for (let runwayPair = 0; runwayPair < runways.length; runwayPair++) {
-            runways[runwayPair][runwayPrimaryEndIndex].removeQueue(aircraft, true);
-            runways[runwayPair][runwaySecondaryEndIndex].removeQueue(aircraft, true);
```
Comment (n8rzz) [inline src/assets/scripts/airport/AirportController.js:204]: What was the `force` value used for previously?  Glad it's not needed now. 👍

**Review by n8rzz — APPROVED** [review]
Seems like there is a conflict with the changelog, but after that this looks good.  Nice work!!

File: src/assets/scripts/airport/AirportController.js (hunk 2 above)
Comment (erikquinn) [reply src/assets/scripts/airport/AirportController.js:204]: I don't remember now, but I remember that when looking at it, I determined that it is no longer a factor. :smile:

**Pull request merged**